src/widget.py

```diff
@@ -1,4 +1,4 @@
-from masks import get_mask_account, get_mask_card_number
+from masks import get_mask_account, get_mask_card_number  # type: ignore[import-not-found]
```

Comment:
Импорт нужно делать из src.masks
```suggestion
from src.masks import get_mask_account, get_mask_card_number
```